File: Domain/Resenia.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain
{
    public class Resenia
    {
        public int Id { get; set; }
        public int Puntaje { get; set; }
        public string Texto { get; set; }
        public virtual DatosUsuario Datos {get;set;}
    }
}

File: SPA/FrontEnd/src/app/Models/Resenia/resenia.ts
import { DatosUsuario } from '../Datos/DatosUsuario/datos-usuario';
export class Resenia {
    id: number;
    puntaje: number;
    texto: string;
    datos: DatosUsuario;

    constructor(id: number, puntaje: number, texto: string, datos: DatosUsuario) {
        this.id = id;
        this.puntaje = puntaje;
        this.texto = texto;
        this.datos = datos;
    }
}
